app/javascript/components/Filters/FormFilter/__snapshots__/component.test.js.snap

```diff
@@ -1,78 +1,56 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`popover handles callbacks 1`] = `
-[MockFunction] {
-  "calls": Array [
-    Array [],
-  ],
-  "results": Array [
-    Object {
-      "type": "return",
-      "value": undefined,
-    },
-  ],
-}
+exports[`popover handles callbacks 1`] = `1`;
+
+exports[`popover handles callbacks 2`] = `
```

Comment:
These two snapshots were split up because now we get the actual `event` which has a MASSIVE stringified value. So it's logging the number of calls instead of the whole `calls` array now.